less/hotgrid.less

```diff
@@ -7,7 +7,7 @@
   }
 
   &__item {
-    width: 50%;
+    width: var(--adapt-hotgrid-item-width);
```

Comment:
Won't this change the item width for all Hot Grids on the page? So, you would not be able to use 3 and 4 column Hot Grids on the same page?